src/main/java/com/aws/greengrass/disk/spool/DiskSpoolDAO.java

```diff
@@ -82,10 +89,14 @@ public DiskSpoolDAO(NucleusPaths paths) throws IOException {
      * @throws SQLException if db is unable to be created
      */
     public void initialize() throws SQLException {
+        DriverManager.registerDriver(new org.sqlite.JDBC());
```

Comment:
do we need to register a new instance of a driver every time we make a new connection? Sounds unnecessary

Reply:
It's unnecessary but it also doesn't have an effect according to the docs, would need to see how much extra work it is or isn't doing

Reply:
Feels like a leak given you create a new driver each time.

Reply:
Yeah will look more closely, this is also existing behavior 🙃

Reply:
so turns out this line isn't even needed, driver will self-register anyway, removed it

Reply:
> so turns out this line isn't even needed, driver will self-register anyway, removed it

this was wrong, not sure why it was working in local testing, but it failed UATs saying that driver's not found
```
java.io.IOException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:/home/ec2-user/testResults/23a8fe84c2b77e4bcf9352b2925df0db9fd6d40b/work/aws.greengrass.DiskSpooler/spooler.db
```
Will add back in